opsmngr/automation_config.go

```diff
@@ -344,7 +345,7 @@ type Process struct {
 	LogRotate                                  *LogRotate         `json:"logRotate,omitempty"`
 	ManualMode                                 bool               `json:"manualMode"`
 	Name                                       string             `json:"name,omitempty"`
-	NumCores                                   int                `json:"numCores"`
+	NumCores                                   int                `json:"numCores,omitempty"`
```

Comment:
if zero is not a valid value this is ok, if not we may need to change this to a pointer

Reply:
As I understood `0` is a valid value but it won’t be considered within mongod - https://github.com/mongodb/mongo/blob/2931f3c764f8e093ae31adb4b2b62c1ce01d7421/src/mongo/executor/task_executor_pool.cpp#L51